docs/configuring-playbook-bridge-appservice-irc.md

```diff
@@ -0,0 +1,435 @@
+# Setting up Appservice IRC (optional)
+
+The playbook can install and configure [matrix-appservice-irc](https://github.com/TeDomum/matrix-appservice-irc) for you.
+
+See the project's [documentation](https://github.com/TeDomum/matrix-appservice-irc/blob/master/HOWTO.md) to learn what it does and why it might be useful to you.
+
+You'll need to use the following playbook configuration:
+
+```yaml
+matrix_matrix_appservice_irc_enabled: true
```

Comment:
The `roles/matrix-synapse/defaults/main.yml` file actually uses `matrix_appservice_irc_enabled`, not `matrix_matrix_appservice_irc_enabled`.

I am also in favor of dropping the double `matrix` prefix. We just need to fix this documentation.